spanner/api/Spanner.Samples.Tests/QueryWithParameterAsyncTest.cs

```diff
@@ -30,6 +30,6 @@ public async Task TestQueryWithParameterAsync()
     {
         QueryWithParameterAsyncSample sample = new QueryWithParameterAsyncSample();
         var singers = await sample.QueryWithParameterAsync(_spannerFixture.ProjectId, _spannerFixture.InstanceId, _spannerFixture.DatabaseId);
-        Assert.Contains(singers, s => s.FirstName == "Marc" && s.LastName == "Richards" && s.SingerId == 1);
+        Assert.DoesNotContain(singers, s => s.FirstName == "Melissa" && s.LastName == "Garcia" && s.SingerId == 1);
```

Comment:
This is because the record with first name as `Melissa` and `Garcia` as last name is inserted in `WriteUsingDmlCoreAsyncSample` and test for same is run after `QueryWithParameterAsync`.